CoffeeChat/Controller/EditLocationAvailabilityViewController.swift

```diff
@@ -0,0 +1,296 @@
+//
+//  EditLocationAvailabilityViewController.swift
+//  CoffeeChat
+//
+//  Created by Amy Chin Siu Huang on 11/19/20.
+//  Copyright © 2020 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+// MARK: - Location UICollectionView Header
+private class LocationHeaderLabel: UICollectionReusableView {
```

Comment:
Maybe move this into another folder? I think we should have a `View` folder. And our naming convention would be something like `LocationHeaderLabelView`